### DISCUSSION TITLE
Questions sur addHead

### DISCUSSION THREAD

**Discussion by moimp3** [rawsrc/PhpEcho#31] [General]
Est-ce qu'il est possible avec addHead de prévoir l'emplacement de l'ajout dans la liste des balises de l'entête?
Par exemple, dans le cas de plusieurs fichiers de styles, il peut être nécessaire de placer la feuille de styles prioritaire après les autres.

Désolé, ma question est prématurée. Il suffit de la placer au bon endroit dans le bloc.

**Comment by rawsrc** [comment]
> Il suffit de la placer au bon endroit dans le bloc.

Exactement

@++
